test/common/formatter/command_extension.h

```diff
@@ -35,5 +35,30 @@ class TestCommandFactory : public CommandParserFactory {
   std::string name() const override;
 };
 
+class AdditionalFormatter : public FormatterProvider {
+public:
+  // FormatterProvider
+  absl::optional<std::string> format(const Http::RequestHeaderMap&, const Http::ResponseHeaderMap&,
+                                     const Http::ResponseTrailerMap&, const StreamInfo::StreamInfo&,
+                                     absl::string_view) const override;
+  ProtobufWkt::Value formatValue(const Http::RequestHeaderMap&, const Http::ResponseHeaderMap&,
+                                 const Http::ResponseTrailerMap&, const StreamInfo::StreamInfo&,
+                                 absl::string_view) const override;
+};
+
+class AdditionalCommandParser : public CommandParser {
+public:
+  AdditionalCommandParser() = default;
```

Comment:
dont think this is necessary?

Reply:
some day, I'll get around adding that clang-tidy check that I promised for this